### ISSUE TITLE
/lib/ld-linux.so.2 could not open

### ISSUE THREAD

**Issue by dothzor** [Heroic-Games-Launcher/HeroicGamesLauncher#1555]
Hi, I've been using HGS on my Steam Deck and I can't run any game. 
After installing a game - for example Into The Breach which error log I attach below - and hitting "play" button it runs for a few secs then it stops. During this few secs game doesn't actually start, I'm still in HGS menu.
Tried running it in both desktop mode and gaming mode - same result. Could anybody give me a hint on what I'm doing wrong?

Launch Command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/deck/.var/app/com.heroicgameslauncher.hgl/.steam/steam STEAM_COMPAT_DATA_PATH=/home/deck/Games/Heroic/Prefixes/IntoTheBreach STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-IntoTheBreach PROTON_LOG_DIR=/home/deck/.var/app/com.heroicgameslauncher.hgl /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Blobfish --language en --no-wine --wrapper "'/home/deck/.local/share/Steam/steamapps/common/Proton - Experimental/proton' run"

System Info:
Heroic Version: 2.4.0-beta Caesar Clown
Legendary Version:  0.20.26 Dark Energy
OS: Freedesktop.org KERNEL: 5.13.0-valve21-1-neptune-02209-g2a5bdc1102a0 ARCH: x64
CPU: AMD Custom APU 0405 @1.8 GOVERNOR: schedutil
RAM: Total: 14.49 GiB Available: 11.85 GiB
GRAPHICS: GPU0: Device 163f VRAM: 512MB DRIVER:  
PROTOCOL: x11

Game Settings: {
"autoInstallDxvk": true,
"autoInstallVkd3d": true,
"maxSharpness": 2,
"language": "",
"nvidiaPrime": false,
"otherOptions": "",
"savesPath": "",
"showFps": false,
"targetExe": "",
"useGameMode": false,
"wineCrossoverBottle": "Heroic",
"winePrefix": "/home/deck/Games/Heroic/Prefixes/IntoTheBreach",
"wineVersion": {
"bin": "/home/deck/.local/share/Steam/steamapps/common/Proton - Experimental/proton",
"name": "Proton - Proton - Experimental",
"type": "proton"
}
}

Error Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching Blobfish...
Proton: Upgrading prefix from 6.3-3 to 7.0-100 (/home/deck/Games/Heroic/Prefixes/IntoTheBreach/)
fsync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
WARNING: radv is not a conformant Vulkan implementation, testing use only.
WARNING: radv is not a conformant Vulkan implementation, testing use only.
/lib/ld-linux.so.2: could not open
/lib/ld-linux.so.2: could not open
/lib/ld-linux.so.2: could not open
wine: configuration in L"/home/deck/Games/Heroic/Prefixes/IntoTheBreach/pfx" has been updated.
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API FAIL] SteamAPI_Init() failed
/lib/ld-linux.so.2: could not open

**Comment by Nocccer** [comment]
Remove prefix and disable dxvk and v3dvk off. On proton it should not be possible to turn them on? How you did that. Also if that does not the trick try proton 7.x instead of experimental

**Comment by dothzor** [comment]
Thanks for the tips. Unfortunately since it's my first time with a device on Linux I have no idea how to do any of those. How exactly should I disable dxvk and vdk3d? I don't know how were they enabled - I just downloaded and installed Heroic flat pack and then installed game. Also how do I remove prefix?

Sorry for totally newb questions :/

I tried changing proton to 7.x and 6.x before, it hasn't helped.

**Comment by Nocccer** [comment]
Under Game settings you see the path to the prefix. Delete this folder. Choose Proton 7.0. 
Also make sure that under general settings > wine dxvk and vdk3d is toggled off. Check the same thing under the game settings. Normally if proton is used, this option should not be there, if it is shown it is a bug.

**Comment by Nocccer** [comment]
I also see that you use heroic beta. I only tried 2.3.9 on the deck so far and had no problems.

**Comment by dothzor** [comment]
Thank you very much, it helped. I've reinstalled heroic to v 2.3.9, disabled dxvk and vdk3d and now it's working. I would be stuck there forever without your help. Really appreciate it.

**Comment by Nocccer** [comment]
Interesting. Will test beta then and see if i get the same error.

**Comment by MattyXarope** [comment]
I'm getting this same error on 2.4.3 on Steam Deck

**Comment by imLinguin** [comment]
try installing C 32bit libs
run the following command in a terminal (Konsole is fine):
`flatpak install org.freedesktop.Platform.Compat.i386`

**Comment by MattyXarope** [comment]
> try installing C 32bit libs run the following command in a terminal (Konsole is fine): `flatpak install org.freedesktop.Platform.Compat.i386`

That did get me a bit further;

```System Info:
Heroic Version: 2.4.3 Chopper
Legendary Version:  0.20.28 Dark Energy (hotfix #2)
OS: Freedesktop SDK KERNEL: 5.13.0-valve21.1-1-neptune-02211-gc54cda5a36f3 ARCH: x64
CPU: AMD Custom APU 0405 @1.95 GOVERNOR: schedutil
RAM: Total: 14.49 GiB Available: 11.48 GiB
GRAPHICS: GPU0: VanGogh [AMD Custom GPU 0405] VRAM: 512MB DRIVER:  
PROTOCOL: x11

Game Settings: {
	"autoInstallDxvk": true,
	"autoInstallVkd3d": true,
	"preferSystemLibs": false,
	"maxSharpness": 2,
	"launcherArgs": "-dx11",
	"nvidiaPrime": false,
	"enviromentOptions": [],
	"wrapperOptions": [],
	"savesPath": "",
	"showFps": false,
	"targetExe": "",
	"useGameMode": true,
	"language": "",
	"wineVersion": {
		"bin": "/home/deck/.local/share/Steam/compatibilitytools.d/GE-Proton7-26/proton",
		"name": "Proton - GE-Proton7-26",
		"type": "proton"
	},
	"winePrefix": "/home/deck/.wine",
	"eacRuntime": true,
	"battlEyeRuntime": false,
	"gogSaves": []
}

Game launched at: Wed Sep 14 2022 09:25:43 GMT+0200 (Central European Summer Time)

Launch Command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/deck/.var/app/com.heroicgameslauncher.hgl/.steam/steam STEAM_COMPAT_DATA_PATH=/home/deck/.wine PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 PROTON_EAC_RUNTIME=/home/deck/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/runtimes/eac_runtime STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-EvilDeadTheGame PROTON_LOG_DIR=/home/deck/.var/app/com.heroicgameslauncher.hgl /app/bin/gamemoderun /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch 5c147002780d4e6eb292f54edfe73fda --language en  --no-wine --wrapper "'/home/deck/.local/share/Steam/compatibilitytools.d/GE-Proton7-26/proton' run" -dx11

Game Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching 5c147002780d4e6eb292f54edfe73fda...
gamemodeauto: 
gamemodeauto: 
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API FAIL] SteamAPI_Init() failed
terminate called after throwing an instance of 'dxvk::DxvkError'
gamemodeauto:```

**Comment by Nocccer** [comment]
> > try installing C 32bit libs run the following command in a terminal (Konsole is fine): `flatpak install org.freedesktop.Platform.Compat.i386`
> 
> That did get me a bit further;
> 
> ```System Info:
> Heroic Version: 2.4.3 Chopper
> Legendary Version:  0.20.28 Dark Energy (hotfix #2)
> OS: Freedesktop SDK KERNEL: 5.13.0-valve21.1-1-neptune-02211-gc54cda5a36f3 ARCH: x64
> CPU: AMD Custom APU 0405 @1.95 GOVERNOR: schedutil
> RAM: Total: 14.49 GiB Available: 11.48 GiB
> GRAPHICS: GPU0: VanGogh [AMD Custom GPU 0405] VRAM: 512MB DRIVER:  
> PROTOCOL: x11
> 
> Game Settings: {
> 	"autoInstallDxvk": true,
> 	"autoInstallVkd3d": true,
> 	"preferSystemLibs": false,
> 	"maxSharpness": 2,
> 	"launcherArgs": "-dx11",
> 	"nvidiaPrime": false,
> 	"enviromentOptions": [],
> 	"wrapperOptions": [],
> 	"savesPath": "",
> 	"showFps": false,
> 	"targetExe": "",
> 	"useGameMode": true,
> 	"language": "",
> 	"wineVersion": {
> 		"bin": "/home/deck/.local/share/Steam/compatibilitytools.d/GE-Proton7-26/proton",
> 		"name": "Proton - GE-Proton7-26",
> 		"type": "proton"
> 	},
> 	"winePrefix": "/home/deck/.wine",
> 	"eacRuntime": true,
> 	"battlEyeRuntime": false,
> 	"gogSaves": []
> }
> 
> Game launched at: Wed Sep 14 2022 09:25:43 GMT+0200 (Central European Summer Time)
> 
> Launch Command: STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/deck/.var/app/com.heroicgameslauncher.hgl/.steam/steam STEAM_COMPAT_DATA_PATH=/home/deck/.wine PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 PROTON_EAC_RUNTIME=/home/deck/.var/app/com.heroicgameslauncher.hgl/config/heroic/tools/runtimes/eac_runtime STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-EvilDeadTheGame PROTON_LOG_DIR=/home/deck/.var/app/com.heroicgameslauncher.hgl /app/bin/gamemoderun /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch 5c147002780d4e6eb292f54edfe73fda --language en  --no-wine --wrapper "'/home/deck/.local/share/Steam/compatibilitytools.d/GE-Proton7-26/proton' run" -dx11
> 
> Game Log:
> [cli] INFO: Logging in...
> [Core] INFO: Trying to re-use existing login session...
> [cli] INFO: Checking for updates...
> [Core] INFO: Getting authentication token...
> [cli] INFO: Launching 5c147002780d4e6eb292f54edfe73fda...
> gamemodeauto: 
> gamemodeauto: 
> wineserver: using server-side synchronization.
> wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
> [S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
> [S_API FAIL] SteamAPI_Init() failed
> terminate called after throwing an instance of 'dxvk::DxvkError'
> gamemodeauto:```

Someone had also this error. He solved it by reinstall heroic.
